pkg/testutils/feature_flag.go

```diff
@@ -16,3 +17,13 @@ func RunWithFeatureFlagEnabled(t *testing.T, flag features.FeatureFlag, subTest
 
 	subTest(t)
 }
+
+// MustUpdateFeature will attempt to set the feature flag to the desired value,
+// if unable to do so will skip the test.
+func MustUpdateFeature(t *testing.T, flag features.FeatureFlag, desired bool) {
```

Comment:
nit: maybe also refactor `RunWithFeatureFlagEnabled` to re-use this function here?